test/integration/user-management-api/application_plan_features_test.rb

```diff
@@ -105,10 +105,11 @@ def teardown
 
     test 'associate existing feature' do
```

Comment:
Actually, after changing the test the test name is not correct anymore, because the test is not associating the existing feature.

I guess it should probably try to post the same `feat`, and assert that count returns 1. Or additionally assert that response is 409, like we did with service plan features test.

Reply:
Done!